src/components/shortcircuit-analysis-result.js

```diff
@@ -6,30 +6,127 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactJson from 'react-json-view';
+import { useIntl } from 'react-intl';
 import { useSelector } from 'react-redux';
-import { PARAM_THEME } from '../utils/config-params';
-import { LIGHT_THEME } from '@gridsuite/commons-ui';
+import VirtualizedTable from './util/virtualized-table';
 
 const ShortCircuitAnalysisResult = ({ result }) => {
-    const selectedTheme = useSelector((state) => state[PARAM_THEME]);
+    const intl = useIntl();
 
     const shortCircuitNotif = useSelector((state) => state.shortCircuitNotif);
 
+    function flattenResult(shortcutAnalysisResult) {
```

Comment:
I'll keep this function in the component because it is specific to it. I don't think splitting the function into two will improve the readability of the code.